From 0c68a6ec67961f17daadb4dcd6ae3eb249f27193 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Tue, 15 Jan 2019 16:56:51 -0800
Subject: [PATCH 2277/2278] net: sched: sch_taprio file needed to be updated
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

too many arguments to function ‘qdisc_create_dflt’
hence need to fix the argument following by definition

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 net/sched/sch_taprio.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/net/sched/sch_taprio.c b/net/sched/sch_taprio.c
index 1754c75..51b47dd 100644
--- a/net/sched/sch_taprio.c
+++ b/net/sched/sch_taprio.c
@@ -633,8 +633,7 @@ static int taprio_change(struct Qdisc *sch, struct nlattr *opt,
 		qdisc = qdisc_create_dflt(dev_queue,
 					  &pfifo_qdisc_ops,
 					  TC_H_MAKE(TC_H_MAJ(sch->handle),
-						    TC_H_MIN(i + 1)),
-					  extack);
+						    TC_H_MIN(i + 1)));
 		if (!qdisc)
 			return -ENOMEM;
 
-- 
2.7.4

